app/screens/home/channel_list/categories_list/categories/categories.tsx

```diff
@@ -49,12 +49,19 @@ const Categories = ({
     const isTablet = useIsTablet();
     const switchingTeam = useTeamSwitch();
     const teamId = categories[0]?.teamId;
+    const showAllCategory = onlyUnreads && !unreadsOnTop;
```

Comment:
After doing some testing locally, this variable should probably called "showOnlyUnreadsCategory". With that name everything will be more clear.

Reply:
@larkox Updated